lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +217,49 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        result_list = []
+        sent_list = split_by_sentence(text)
+        for position, element in enumerate(sent_list):
+            conllu_tokens = []
+            words = ConlluToken(element).get_cleaned().split()
+
+            for one_word in words:
+                conllu_tokens.append(ConlluToken(one_word))
+
+            result_list.append(ConlluSentence(position=position,
+                                              text=element,
+                                              tokens=conllu_tokens))
+        return result_list
 
     def run(self) -> None:
         """
         Performs basic preprocessing and writes processed text to files
         """
+        art_dict = self._corpus.get_articles()
+
+        for one_art in art_dict.values():
+            art_id = one_art.article_id
+            name = art_id + '_raw.txt'
+            path_raw = self._corpus.path / name
+
+            article_raw = from_raw(path_raw, one_art)
+            raw_txt = article_raw.get_raw_text()
+
+            conllu_sentences = self._process(raw_txt)
+            article_raw.set_conllu_sentences(conllu_sentences)
+
+            cleaned_name = art_id + '_cleaned.txt'
+            cleaned_path = self._corpus.path / cleaned_name
+
+            with open(cleaned_path, 'w', encoding='utf-8') as f:
```

Comment:
I do not see creation of conllu files (`to_conllu` call)

Reply:
i'm on my way to it..